stable/appmesh-controller/values.yaml

```diff
@@ -15,7 +15,7 @@ image:
 sidecar:
```

Comment:
I don't see the default controller image updated to v1.4.0. We need to update it.